ue4docker/infrastructure/BuildConfiguration.py

```diff
@@ -55,7 +57,8 @@ class VisualStudio(object):
         #
         # Unreal Engine 4.25 is the first that works with .NET SDK 4.7+
         # See https://github.com/EpicGames/UnrealEngine/commit/5256eedbdef30212ab69fdf4c09e898098959683
-        VS2019: semver.VersionInfo(4, 25)
+        VS2019: semver.VersionInfo(4, 25),
+        VS2022: semver.VersionInfo(4, 25),
```

Comment:
I believe this should be something more like 4.27? Ideally, this should be tested to figure out exact release where this started to work.

Reply:
See original comment. I agree it should be tested, but I can't do all those full builds at this moment.